includes/data-stores/class-wc-admin-reports-data-store.php

```diff
@@ -634,6 +634,65 @@ protected function get_excluded_coupons( $query_args ) {
 		return $excluded_coupons_str;
 	}
 
+	/**
+	 * Returns comma separated ids of included orders, based on query arguments from the user.
+	 *
+	 * @param array $query_args Parameters supplied by the user.
+	 * @return string
+	 */
+	protected function get_included_orders( $query_args ) {
+		$included_orders_str = '';
+
+		if ( isset( $query_args['order_includes'] ) && is_array( $query_args['order_includes'] ) && count( $query_args['order_includes'] ) > 0 ) {
+			$included_orders_str = implode( ',', $query_args['order_includes'] );
```

Comment:
do we need to cast/ensure each value is an int here?

Reply:
Same comment on all of the below `get_` methods?

Reply:
Correct - the ID ones get ran through `absint` from it: https://developer.wordpress.org/reference/functions/wp_parse_id_list/